### DISCUSSION TITLE
Spotube on xbox?

### DISCUSSION THREAD

**Discussion by SKaterTux** [KRTirtho/spotube#702] [General]
I have cancelled my spotify subscription due to this app and it works on everything apart from xbox, there is no app for it. so if this could be added that would be amazing

**Comment by KRTirtho** [comment]
Yea that's interesting but I'm unaware if Flutter supports xbox

**Comment by fxzxmicah** [comment]
Should we first need to package it in Windows MSIX format?

**Reply by KRTirtho** [reply]
Believe me that's the easiest part.
We'll need a new embedder for Xbox as it doesn't support Win32 APIs

**Reply by fxzxmicah** [reply]
Yes, but Windows can also use MSIX package, so will there be MSIX package for Windows? Released through Microsoft Store.

**Reply by KRTirtho** [reply]
Yeah that can be done later. But we've first make sure the app runs on  the platform which it doesn't:sweat_smile:

**Reply by fxzxmicah** [reply]
I believe you can do it together.
